buidl/script.py

```diff
@@ -366,6 +367,36 @@ def convert(cls, raw_redeem_script):
         stream = BytesIO(encode_varstr(raw_redeem_script))
         return cls.parse(stream)
 
+    @classmethod
+    def create_p2sh_multisig(cls, quorum_m, pubkey_hex_list, sort_keys=True):
+        """
+        Create a p2sh RedeemScript using a configure threshold (quorum_m) of child public keys (in hex).
+
+        To use a custom order of pubkeys, feed them in order and set sort_keys=False
+        """
+        # safety checks
+        if type(quorum_m) is not int:
+            raise ValueError(f"quorum_m must be of type int: {quorum_m}")
+        if quorum_m < 1 or quorum_m > len(pubkey_hex_list):
+            raise ValueError(f"Invalid m-of-n: {quorum_m}-of-{len(pubkey_hex_list)}")
+
+        commands = [OP_CODE_NAMES_LOOKUP[f"OP_{quorum_m}"]]
```

Comment:
there's a helper function `number_to_op_code_byte` which does this instead of having to use the lookup.